src/collapse/perf.rs

```diff
@@ -72,6 +84,10 @@ struct PerfState {
     /// Number of times each call stack has been seen.
     occurrences: HashMap<String, usize>,
 
+    /// Cached Contexts by module name for un_inline() since constructing them is somewhat costly.
+    addr2line_contexts:
+        HashMap<String, addr2line::Context<gimli::EndianRcSlice<gimli::RunTimeEndian>>>,
```

Comment:
Watch https://github.com/gimli-rs/addr2line/issues/109 for whether we can simplify this